e2e/tests/app.e2e.js

```diff
@@ -0,0 +1,65 @@
+import { expect, test } from '@playwright/test';
+import { HomePage } from '../pages/home-page';
+import { NavBarPage } from '../pages/nav-bar-page';
+
+test.describe('AngularJS-Material-Vite app', () => {
+  let errorLogs;
```

Comment:
```suggestion
  let errorLogs: string[];
```

Reply:
This repository does not have TypeScript in its dependencies 😄
<img width="540" alt="image" src="https://user-images.githubusercontent.com/5073455/224045960-f4cc872c-db90-4a1d-b1fe-23fa20678c18.png">

Reply:
Ooops! My bad 😇